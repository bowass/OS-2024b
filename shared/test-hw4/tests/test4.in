42169
52162
54136
3101
15794
5702
62123
39941
1455
60403
22217
38864
61710
37056
62277
61639
38101
59853
37191
9440
5351
35685
30921
55472
33767
65387
6891
40054
50944
52097
35312
63698
37668
19119
4218
51888
64581
13927
37504
10140
6038
27391
41862
61546
56853
50027
28791
22176
36804
62770
42423
50731
18757
46862
9074
42526
24872
32849
32514
20294
29848
38641
13848
41952
7916
33017
52945
59171
23229
974
26742
45464
10402
29370
21927
55419
65374
46331
23996
32881
33786
46069
13579
53790
13962
11648
2120
15641
49645
41691
8909
26488
22943
56218
29661
29752
32298
35757
58052
900
7693
20023
7668
5068
23696
11302
7094
34699
20362
47238
26987
55732
38768
49623
55207
43053
21380
47653
45943
2270
63470
13030
36230
21230
30676
54810
48702
19226
45096
53454
64847
43520
58751
34361
62669
26269
17083
42910
5533
5053
12237
6466
28552
62592
12122
53842
58295
52562
44166
57037
3840
51207
25597
32500
53995
57589
54669
36582
19619
34941
61663
20829
10659
34772
33322
44921
2073
15845
53882
24357
6050
53090
46125
63660
54188
4629
60629
19360
60707
54758
41442
30456
585
2308
62469
34233
13075
3222
6317
45064
29097
10296
43339
31116
39809
42891
8418
21145
50208
12162
64251
17690
5915
51360
3971
11539
9724
62833
17304
43795
49249
29796
11955
23171
30119
13453
63710
14060
17000
19711
44047
63907
41429
49444
16531
58006
23400
15139
35691
41771
47815
12377
54223
35812
33843
14450
34235
62167
34230
8879
28718
34583
30419
15919
25048
19323
16148
38886
43540
57051
22918
49204
44152
46298
2557
36223
55272
15332
18826
36356
46613
19160
61719
6497
51443
44548
24623
52923
39337
43266
20392
6651
59938
26305
2925
29308
5947
37272
44504
62486
18723
34094
58980
5322
60169
23736
58268
14144
63899
18930
42695
30202
13249
22096
65090
48493
29379
21809
49167
47892
64283
26013
25817
29792
10349
44904
35042
43640
5173
52486
5331
28951
57551
54369
24858
40163
26963
41611
18943
14427
20288
39021
37430
59360
51551
42975
3753
1013
179
39277
50198
28521
28632
18150
1389
25473
39866
43374
33209
23019
18712
30470
25279
22274
50251
4936
20560
53927
21648
48701
44444
40230
8321
49957
58063
34231
62465
25571
62356
2463
58112
26290
63683
29024
15467
10349
9196
3159
30006
43208
8552
58134
45479
21158
18616
25412
35824
47764
45201
27897
50347
40757
32178
22017
5700
26952
19046
16413
36744
42303
32876
44454
34013
2940
12731
3925
39499
43209
19591
28500
8726
54213
29604
48003
37208
21539
57991
11161
11526
43461
7100
53725
51532
963
2705
10674
11956
54311
51107
28700
11458
63795
443
51614
36214
36409
48054
34204
31811
13488
22863
3466
38185
47127
16760
28977
6092
37300
31281
14555
21671
29921
4796
46655
24440
58885
49677
63906
24692
53968
21680
42910
33586
9541
59799
51566
22899
7642
8969
61612
14542
468
50595
2333
58085
51177
35854
41599
55685
10048
46318
23789
56542
11851
4441
48034
55853
17870
3696
43947
63797
11346
55231
8667
44441
44408
37102
22436
30161
23183
20250
24693
20395
22939
23414
7299
27911
64246
17120
57800