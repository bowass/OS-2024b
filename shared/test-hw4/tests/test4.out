Virtual address: 42169 Physical address: 185 Value: 0
Virtual address: 52162 Physical address: 450 Value: 50
Virtual address: 54136 Physical address: 632 Value: 0
Virtual address: 3101 Physical address: 797 Value: 0
Virtual address: 15794 Physical address: 1202 Value: 15
Virtual address: 5702 Physical address: 1350 Value: 5
Virtual address: 62123 Physical address: 1707 Value: -86
Virtual address: 39941 Physical address: 1797 Value: 0
Virtual address: 1455 Physical address: 2223 Value: 107
Virtual address: 60403 Physical address: 2547 Value: -4
Virtual address: 22217 Physical address: 2761 Value: 0
Virtual address: 38864 Physical address: 3024 Value: 0
Virtual address: 61710 Physical address: 3086 Value: 60
Virtual address: 37056 Physical address: 3520 Value: 0
Virtual address: 62277 Physical address: 3653 Value: 0
Virtual address: 61639 Physical address: 4039 Value: 49
Virtual address: 38101 Physical address: 4309 Value: 0
Virtual address: 59853 Physical address: 4557 Value: 0
Virtual address: 37191 Physical address: 4679 Value: 81
Virtual address: 9440 Physical address: 5088 Value: 0
Virtual address: 5351 Physical address: 5351 Value: 57
Virtual address: 35685 Physical address: 5477 Value: 0
Virtual address: 30921 Physical address: 5833 Value: 0
Virtual address: 55472 Physical address: 6064 Value: 0
Virtual address: 33767 Physical address: 6375 Value: -7
Virtual address: 65387 Physical address: 6507 Value: -38
Virtual address: 6891 Physical address: 6891 Value: -70
Virtual address: 40054 Physical address: 1910 Value: 39
Virtual address: 50944 Physical address: 6912 Value: 0
Virtual address: 52097 Physical address: 385 Value: 0
Virtual address: 35312 Physical address: 7408 Value: 0
Virtual address: 63698 Physical address: 7634 Value: 62
Virtual address: 37668 Physical address: 7716 Value: 0
Virtual address: 19119 Physical address: 8111 Value: -85
Virtual address: 4218 Physical address: 8314 Value: 4
Virtual address: 51888 Physical address: 8624 Value: 0
Virtual address: 64581 Physical address: 8773 Value: 0
Virtual address: 13927 Physical address: 9063 Value: -103
Virtual address: 37504 Physical address: 9344 Value: 0
Virtual address: 10140 Physical address: 9628 Value: 0
Virtual address: 6038 Physical address: 9878 Value: 5
Virtual address: 27391 Physical address: 10239 Value: -65
Virtual address: 41862 Physical address: 10374 Value: 40
Virtual address: 61546 Physical address: 3946 Value: 60
Virtual address: 56853 Physical address: 10517 Value: 0
Virtual address: 50027 Physical address: 10859 Value: -38
Virtual address: 28791 Physical address: 11127 Value: 29
Virtual address: 22176 Physical address: 2720 Value: 0
Virtual address: 36804 Physical address: 11460 Value: 0
Virtual address: 62770 Physical address: 11570 Value: 61
Virtual address: 42423 Physical address: 11959 Value: 109
Virtual address: 50731 Physical address: 12075 Value: -118
Virtual address: 18757 Physical address: 12357 Value: 0
Virtual address: 46862 Physical address: 12558 Value: 45
Virtual address: 9074 Physical address: 12914 Value: 8
Virtual address: 42526 Physical address: 13086 Value: 41
Virtual address: 24872 Physical address: 13352 Value: 0
Virtual address: 32849 Physical address: 13649 Value: 0
Virtual address: 32514 Physical address: 13826 Value: 31
Virtual address: 20294 Physical address: 14150 Value: 19
Virtual address: 29848 Physical address: 14488 Value: 0
Virtual address: 38641 Physical address: 14833 Value: 0
Virtual address: 13848 Physical address: 8984 Value: 0
Virtual address: 41952 Physical address: 10464 Value: 0
Virtual address: 7916 Physical address: 15084 Value: 0
Virtual address: 33017 Physical address: 13817 Value: 0
Virtual address: 52945 Physical address: 15313 Value: 0
Virtual address: 59171 Physical address: 15395 Value: -56
Virtual address: 23229 Physical address: 15805 Value: 0
Virtual address: 974 Physical address: 16078 Value: 0
Virtual address: 26742 Physical address: 16246 Value: 26
Virtual address: 45464 Physical address: 16536 Value: 0
Virtual address: 10402 Physical address: 16802 Value: 10
Virtual address: 29370 Physical address: 17082 Value: 28
Virtual address: 21927 Physical address: 17319 Value: 105
Virtual address: 55419 Physical address: 6011 Value: 30
Virtual address: 65374 Physical address: 6494 Value: 63
Virtual address: 46331 Physical address: 17659 Value: 62
Virtual address: 23996 Physical address: 17852 Value: 0
Virtual address: 32881 Physical address: 13681 Value: 0
Virtual address: 33786 Physical address: 6394 Value: 32
Virtual address: 46069 Physical address: 18165 Value: 0
Virtual address: 13579 Physical address: 18187 Value: 66
Virtual address: 53790 Physical address: 18462 Value: 52
Virtual address: 13962 Physical address: 9098 Value: 13
Virtual address: 11648 Physical address: 18816 Value: 0
Virtual address: 2120 Physical address: 19016 Value: 0
Virtual address: 15641 Physical address: 1049 Value: 0
Virtual address: 49645 Physical address: 19437 Value: 0
Virtual address: 41691 Physical address: 19675 Value: -74
Virtual address: 8909 Physical address: 19917 Value: 0
Virtual address: 26488 Physical address: 20088 Value: 0
Virtual address: 22943 Physical address: 20383 Value: 103
Virtual address: 56218 Physical address: 20634 Value: 54
Virtual address: 29661 Physical address: 20957 Value: 0
Virtual address: 29752 Physical address: 14392 Value: 0
Virtual address: 32298 Physical address: 21034 Value: 31
Virtual address: 35757 Physical address: 5549 Value: 0
Virtual address: 58052 Physical address: 21444 Value: 0
Virtual address: 900 Physical address: 16004 Value: 0
Virtual address: 7693 Physical address: 14861 Value: 0
Virtual address: 20023 Physical address: 21559 Value: -115
Virtual address: 7668 Physical address: 22004 Value: 0
Virtual address: 5068 Physical address: 22220 Value: 0
Virtual address: 23696 Physical address: 22416 Value: 0
Virtual address: 11302 Physical address: 22566 Value: 11
Virtual address: 7094 Physical address: 22966 Value: 6
Virtual address: 34699 Physical address: 23179 Value: -30
Virtual address: 20362 Physical address: 14218 Value: 19
Virtual address: 47238 Physical address: 23430 Value: 46
Virtual address: 26987 Physical address: 23659 Value: 90
Virtual address: 55732 Physical address: 23988 Value: 0
Virtual address: 38768 Physical address: 2928 Value: 0
Virtual address: 49623 Physical address: 19415 Value: 117
Virtual address: 55207 Physical address: 24231 Value: -23
Virtual address: 43053 Physical address: 24365 Value: 0
Virtual address: 21380 Physical address: 24708 Value: 0
Virtual address: 47653 Physical address: 24869 Value: 0
Virtual address: 45943 Physical address: 18039 Value: -35
Virtual address: 2270 Physical address: 19166 Value: 2
Virtual address: 63470 Physical address: 25326 Value: 61
Virtual address: 13030 Physical address: 25574 Value: 12
Virtual address: 36230 Physical address: 25734 Value: 35
Virtual address: 21230 Physical address: 26094 Value: 20
Virtual address: 30676 Physical address: 26324 Value: 0
Virtual address: 54810 Physical address: 26394 Value: 53
Virtual address: 48702 Physical address: 26686 Value: 47
Virtual address: 19226 Physical address: 26906 Value: 18
Virtual address: 45096 Physical address: 27176 Value: 0
Virtual address: 53454 Physical address: 27598 Value: 52
Virtual address: 64847 Physical address: 27727 Value: 83
Virtual address: 43520 Physical address: 27904 Value: 0
Virtual address: 58751 Physical address: 28287 Value: 95
Virtual address: 34361 Physical address: 28473 Value: 0
Virtual address: 62669 Physical address: 28877 Value: 0
Virtual address: 26269 Physical address: 29085 Value: 0
Virtual address: 17083 Physical address: 29371 Value: -82
Virtual address: 42910 Physical address: 29598 Value: 41
Virtual address: 5533 Physical address: 29853 Value: 0
Virtual address: 5053 Physical address: 22205 Value: 0
Virtual address: 12237 Physical address: 30157 Value: 0
Virtual address: 6466 Physical address: 30274 Value: 6
Virtual address: 28552 Physical address: 30600 Value: 0
Virtual address: 62592 Physical address: 28800 Value: 0
Virtual address: 12122 Physical address: 30042 Value: 11
Virtual address: 53842 Physical address: 18514 Value: 52
Virtual address: 58295 Physical address: 30903 Value: -19
Virtual address: 52562 Physical address: 31058 Value: 51
Virtual address: 44166 Physical address: 31366 Value: 43
Virtual address: 57037 Physical address: 10701 Value: 0
Virtual address: 3840 Physical address: 31488 Value: 0
Virtual address: 51207 Physical address: 31751 Value: 1
Virtual address: 25597 Physical address: 32253 Value: 0
Virtual address: 32500 Physical address: 21236 Value: 0
Virtual address: 53995 Physical address: 18667 Value: -70
Virtual address: 57589 Physical address: 32501 Value: 0
Virtual address: 54669 Physical address: 32653 Value: 0
Virtual address: 36582 Physical address: 32998 Value: 35
Virtual address: 19619 Physical address: 33187 Value: 40
Virtual address: 34941 Physical address: 33405 Value: 0
Virtual address: 61663 Physical address: 4063 Value: 55
Virtual address: 20829 Physical address: 33629 Value: 0
Virtual address: 10659 Physical address: 33955 Value: 104
Virtual address: 34772 Physical address: 23252 Value: 0
Virtual address: 33322 Physical address: 34090 Value: 32
Virtual address: 44921 Physical address: 34425 Value: 0
Virtual address: 2073 Physical address: 18969 Value: 0
Virtual address: 15845 Physical address: 1253 Value: 0
Virtual address: 53882 Physical address: 18554 Value: 52
Virtual address: 24357 Physical address: 34597 Value: 0
Virtual address: 6050 Physical address: 9890 Value: 5
Virtual address: 53090 Physical address: 34914 Value: 51
Virtual address: 46125 Physical address: 17453 Value: 0
Virtual address: 63660 Physical address: 7596 Value: 0
Virtual address: 54188 Physical address: 684 Value: 0
Virtual address: 4629 Physical address: 35093 Value: 0
Virtual address: 60629 Physical address: 35541 Value: 0
Virtual address: 19360 Physical address: 27040 Value: 0
Virtual address: 60707 Physical address: 35619 Value: 72
Virtual address: 54758 Physical address: 32742 Value: 53
Virtual address: 41442 Physical address: 36066 Value: 40
Virtual address: 30456 Physical address: 36344 Value: 0
Virtual address: 585 Physical address: 36425 Value: 0
Virtual address: 2308 Physical address: 36612 Value: 0
Virtual address: 62469 Physical address: 28677 Value: 0
Virtual address: 34233 Physical address: 37049 Value: 0
Virtual address: 13075 Physical address: 37139 Value: -60
Virtual address: 3222 Physical address: 918 Value: 3
Virtual address: 6317 Physical address: 37549 Value: 0
Virtual address: 45064 Physical address: 27144 Value: 0
Virtual address: 29097 Physical address: 37801 Value: 0
Virtual address: 10296 Physical address: 16696 Value: 0
Virtual address: 43339 Physical address: 37963 Value: 82
Virtual address: 31116 Physical address: 38284 Value: 0
Virtual address: 39809 Physical address: 38529 Value: 0
Virtual address: 42891 Physical address: 29579 Value: -30
Virtual address: 8418 Physical address: 38882 Value: 8
Virtual address: 21145 Physical address: 26009 Value: 0
Virtual address: 50208 Physical address: 38944 Value: 0
Virtual address: 12162 Physical address: 30082 Value: 11
Virtual address: 64251 Physical address: 39419 Value: -66
Virtual address: 17690 Physical address: 39450 Value: 17
Virtual address: 5915 Physical address: 9755 Value: -58
Virtual address: 51360 Physical address: 31904 Value: 0
Virtual address: 3971 Physical address: 31619 Value: -32
Virtual address: 11539 Physical address: 18707 Value: 68
Virtual address: 9724 Physical address: 39932 Value: 0
Virtual address: 62833 Physical address: 11633 Value: 0
Virtual address: 17304 Physical address: 40088 Value: 0
Virtual address: 43795 Physical address: 40211 Value: -60
Virtual address: 49249 Physical address: 40545 Value: 0
Virtual address: 29796 Physical address: 14436 Value: 0
Virtual address: 11955 Physical address: 40883 Value: -84
Virtual address: 23171 Physical address: 15747 Value: -96
Virtual address: 30119 Physical address: 41127 Value: 105
Virtual address: 13453 Physical address: 41357 Value: 0
Virtual address: 63710 Physical address: 7646 Value: 62
Virtual address: 14060 Physical address: 9196 Value: 0
Virtual address: 17000 Physical address: 29288 Value: 0
Virtual address: 19711 Physical address: 33279 Value: 63
Virtual address: 44047 Physical address: 31247 Value: 3
Virtual address: 63907 Physical address: 41635 Value: 104
Virtual address: 41429 Physical address: 36053 Value: 0
Virtual address: 49444 Physical address: 19236 Value: 0
Virtual address: 16531 Physical address: 41875 Value: 36
Virtual address: 58006 Physical address: 21398 Value: 56
Virtual address: 23400 Physical address: 42088 Value: 0
Virtual address: 15139 Physical address: 42275 Value: -56
Virtual address: 35691 Physical address: 5483 Value: -38
Virtual address: 41771 Physical address: 10283 Value: -54
Virtual address: 47815 Physical address: 25031 Value: -79
Virtual address: 12377 Physical address: 42585 Value: 0
Virtual address: 54223 Physical address: 719 Value: -13
Virtual address: 35812 Physical address: 5604 Value: 0
Virtual address: 33843 Physical address: 42803 Value: 12
Virtual address: 14450 Physical address: 43122 Value: 14
Virtual address: 34235 Physical address: 37051 Value: 110
Virtual address: 62167 Physical address: 1751 Value: -75
Virtual address: 34230 Physical address: 37046 Value: 33
Virtual address: 8879 Physical address: 19887 Value: -85
Virtual address: 28718 Physical address: 11054 Value: 28
Virtual address: 34583 Physical address: 23063 Value: -59
Virtual address: 30419 Physical address: 36307 Value: -76
Virtual address: 15919 Physical address: 43311 Value: -117
Virtual address: 25048 Physical address: 13528 Value: 0
Virtual address: 19323 Physical address: 27003 Value: -34
Virtual address: 16148 Physical address: 43540 Value: 0
Virtual address: 38886 Physical address: 3046 Value: 37
Virtual address: 43540 Physical address: 27924 Value: 0
Virtual address: 57051 Physical address: 10715 Value: -74
Virtual address: 22918 Physical address: 20358 Value: 22
Virtual address: 49204 Physical address: 40500 Value: 0
Virtual address: 44152 Physical address: 31352 Value: 0
Virtual address: 46298 Physical address: 17626 Value: 45
Virtual address: 2557 Physical address: 36861 Value: 0
Virtual address: 36223 Physical address: 25727 Value: 95
Virtual address: 55272 Physical address: 24296 Value: 0
Virtual address: 15332 Physical address: 42468 Value: 0
Virtual address: 18826 Physical address: 12426 Value: 18
Virtual address: 36356 Physical address: 32772 Value: 0
Virtual address: 46613 Physical address: 43797 Value: 0
Virtual address: 19160 Physical address: 8152 Value: 0
Virtual address: 61719 Physical address: 3095 Value: 69
Virtual address: 6497 Physical address: 30305 Value: 0
Virtual address: 51443 Physical address: 31987 Value: 60
Virtual address: 44548 Physical address: 44036 Value: 0
Virtual address: 24623 Physical address: 44335 Value: 11
Virtual address: 52923 Physical address: 15291 Value: -82
Virtual address: 39337 Physical address: 44713 Value: 0
Virtual address: 43266 Physical address: 37890 Value: 42
Virtual address: 20392 Physical address: 14248 Value: 0
Virtual address: 6651 Physical address: 30459 Value: 126
Virtual address: 59938 Physical address: 44834 Value: 58
Virtual address: 26305 Physical address: 29121 Value: 0
Virtual address: 2925 Physical address: 45165 Value: 0
Virtual address: 29308 Physical address: 17020 Value: 0
Virtual address: 5947 Physical address: 9787 Value: -50
Virtual address: 37272 Physical address: 4760 Value: 0
Virtual address: 44504 Physical address: 45528 Value: 0
Virtual address: 62486 Physical address: 28694 Value: 61
Virtual address: 18723 Physical address: 12323 Value: 72
Virtual address: 34094 Physical address: 36910 Value: 33
Virtual address: 58980 Physical address: 45668 Value: 0
Virtual address: 5322 Physical address: 5322 Value: 5
Virtual address: 60169 Physical address: 2313 Value: 0
Virtual address: 23736 Physical address: 22456 Value: 0
Virtual address: 58268 Physical address: 30876 Value: 0
Virtual address: 14144 Physical address: 45888 Value: 0
Virtual address: 63899 Physical address: 41627 Value: 102
Virtual address: 18930 Physical address: 12530 Value: 18
Virtual address: 42695 Physical address: 13255 Value: -79
Virtual address: 30202 Physical address: 41210 Value: 29
Virtual address: 13249 Physical address: 37313 Value: 0
Virtual address: 22096 Physical address: 2640 Value: 0
Virtual address: 65090 Physical address: 46146 Value: 63
Virtual address: 48493 Physical address: 46445 Value: 0
Virtual address: 29379 Physical address: 17091 Value: -80
Virtual address: 21809 Physical address: 17201 Value: 0
Virtual address: 49167 Physical address: 40463 Value: 3
Virtual address: 47892 Physical address: 46612 Value: 0
Virtual address: 64283 Physical address: 46875 Value: -58
Virtual address: 26013 Physical address: 47261 Value: 0
Virtual address: 25817 Physical address: 47577 Value: 0
Virtual address: 29792 Physical address: 14432 Value: 0
Virtual address: 10349 Physical address: 16749 Value: 0
Virtual address: 44904 Physical address: 34408 Value: 0
Virtual address: 35042 Physical address: 33506 Value: 34
Virtual address: 43640 Physical address: 28024 Value: 0
Virtual address: 5173 Physical address: 5173 Value: 0
Virtual address: 52486 Physical address: 30982 Value: 51
Virtual address: 5331 Physical address: 5331 Value: 52
Virtual address: 28951 Physical address: 37655 Value: 69
Virtual address: 57551 Physical address: 32463 Value: 51
Virtual address: 54369 Physical address: 47713 Value: 0
Virtual address: 24858 Physical address: 13338 Value: 24
Virtual address: 40163 Physical address: 2019 Value: 56
Virtual address: 26963 Physical address: 23635 Value: 84
Virtual address: 41611 Physical address: 19595 Value: -94
Virtual address: 18943 Physical address: 12543 Value: 127
Virtual address: 14427 Physical address: 43099 Value: 22
Virtual address: 20288 Physical address: 14144 Value: 0
Virtual address: 39021 Physical address: 47981 Value: 0
Virtual address: 37430 Physical address: 9270 Value: 36
Virtual address: 59360 Physical address: 15584 Value: 0
Virtual address: 51551 Physical address: 48223 Value: 87
Virtual address: 42975 Physical address: 29663 Value: -9
Virtual address: 3753 Physical address: 48553 Value: 0
Virtual address: 1013 Physical address: 16117 Value: 0
Virtual address: 179 Physical address: 48819 Value: 44
Virtual address: 39277 Physical address: 44653 Value: 0
Virtual address: 50198 Physical address: 38934 Value: 49
Virtual address: 28521 Physical address: 30569 Value: 0
Virtual address: 28632 Physical address: 30680 Value: 0
Virtual address: 18150 Physical address: 49126 Value: 17
Virtual address: 1389 Physical address: 2157 Value: 0
Virtual address: 25473 Physical address: 32129 Value: 0
Virtual address: 39866 Physical address: 38586 Value: 38
Virtual address: 43374 Physical address: 37998 Value: 42
Virtual address: 33209 Physical address: 49337 Value: 0
Virtual address: 23019 Physical address: 20459 Value: 122
Virtual address: 18712 Physical address: 12312 Value: 0
Virtual address: 30470 Physical address: 26118 Value: 29
Virtual address: 25279 Physical address: 49599 Value: -81
Virtual address: 22274 Physical address: 49666 Value: 21
Virtual address: 50251 Physical address: 38987 Value: 18
Virtual address: 4936 Physical address: 22088 Value: 0
Virtual address: 20560 Physical address: 50000 Value: 0
Virtual address: 53927 Physical address: 18599 Value: -87
Virtual address: 21648 Physical address: 50320 Value: 0
Virtual address: 48701 Physical address: 26685 Value: 0
Virtual address: 44444 Physical address: 45468 Value: 0
Virtual address: 40230 Physical address: 50470 Value: 39
Virtual address: 8321 Physical address: 38785 Value: 0
Virtual address: 49957 Physical address: 10789 Value: 0
Virtual address: 58063 Physical address: 21455 Value: -77
Virtual address: 34231 Physical address: 37047 Value: 109
Virtual address: 62465 Physical address: 28673 Value: 0
Virtual address: 25571 Physical address: 32227 Value: -8
Virtual address: 62356 Physical address: 3732 Value: 0
Virtual address: 2463 Physical address: 36767 Value: 103
Virtual address: 58112 Physical address: 30720 Value: 0
Virtual address: 26290 Physical address: 29106 Value: 25
Virtual address: 63683 Physical address: 7619 Value: 48
Virtual address: 29024 Physical address: 37728 Value: 0
Virtual address: 15467 Physical address: 50795 Value: 26
Virtual address: 10349 Physical address: 16749 Value: 0
Virtual address: 9196 Physical address: 13036 Value: 0
Virtual address: 3159 Physical address: 855 Value: 21
Virtual address: 30006 Physical address: 41014 Value: 29
Virtual address: 43208 Physical address: 24520 Value: 0
Virtual address: 8552 Physical address: 51048 Value: 0
Virtual address: 58134 Physical address: 30742 Value: 56
Virtual address: 45479 Physical address: 16551 Value: 105
Virtual address: 21158 Physical address: 26022 Value: 20
Virtual address: 18616 Physical address: 51384 Value: 0
Virtual address: 25412 Physical address: 32068 Value: 0
Virtual address: 35824 Physical address: 5616 Value: 0
Virtual address: 47764 Physical address: 24980 Value: 0
Virtual address: 45201 Physical address: 27281 Value: 0
Virtual address: 27897 Physical address: 51705 Value: 0
Virtual address: 50347 Physical address: 39083 Value: 42
Virtual address: 40757 Physical address: 51765 Value: 0
Virtual address: 32178 Physical address: 52146 Value: 31
Virtual address: 22017 Physical address: 2561 Value: 0
Virtual address: 5700 Physical address: 1348 Value: 0
Virtual address: 26952 Physical address: 23624 Value: 0
Virtual address: 19046 Physical address: 8038 Value: 18
Virtual address: 16413 Physical address: 41757 Value: 0
Virtual address: 36744 Physical address: 11400 Value: 0
Virtual address: 42303 Physical address: 11839 Value: 79
Virtual address: 32876 Physical address: 13676 Value: 0
Virtual address: 44454 Physical address: 45478 Value: 43
Virtual address: 34013 Physical address: 42973 Value: 0
Virtual address: 2940 Physical address: 45180 Value: 0
Virtual address: 12731 Physical address: 52411 Value: 110
Virtual address: 3925 Physical address: 31573 Value: 0
Virtual address: 39499 Physical address: 52555 Value: -110
Virtual address: 43209 Physical address: 24521 Value: 0
Virtual address: 19591 Physical address: 33159 Value: 33
Virtual address: 28500 Physical address: 30548 Value: 0
Virtual address: 8726 Physical address: 19734 Value: 8
Virtual address: 54213 Physical address: 709 Value: 0
Virtual address: 29604 Physical address: 20900 Value: 0
Virtual address: 48003 Physical address: 46723 Value: -32
Virtual address: 37208 Physical address: 4696 Value: 0
Virtual address: 21539 Physical address: 50211 Value: 8
Virtual address: 57991 Physical address: 21383 Value: -95
Virtual address: 11161 Physical address: 52889 Value: 0
Virtual address: 11526 Physical address: 18694 Value: 11
Virtual address: 43461 Physical address: 38085 Value: 0
Virtual address: 7100 Physical address: 22972 Value: 0
Virtual address: 53725 Physical address: 53213 Value: 0
Virtual address: 51532 Physical address: 48204 Value: 0
Virtual address: 963 Physical address: 16067 Value: -16
Virtual address: 2705 Physical address: 53393 Value: 0
Virtual address: 10674 Physical address: 33970 Value: 10
Virtual address: 11956 Physical address: 40884 Value: 0
Virtual address: 54311 Physical address: 47655 Value: 9
Virtual address: 51107 Physical address: 7075 Value: -24
Virtual address: 28700 Physical address: 11036 Value: 0
Virtual address: 11458 Physical address: 22722 Value: 11
Virtual address: 63795 Physical address: 41523 Value: 76
Virtual address: 443 Physical address: 53691 Value: 110
Virtual address: 51614 Physical address: 48286 Value: 50
Virtual address: 36214 Physical address: 25718 Value: 35
Virtual address: 36409 Physical address: 32825 Value: 0
Virtual address: 48054 Physical address: 46774 Value: 46
Virtual address: 34204 Physical address: 37020 Value: 0
Virtual address: 31811 Physical address: 53827 Value: 16
Virtual address: 13488 Physical address: 41392 Value: 0
Virtual address: 22863 Physical address: 20303 Value: 83
Virtual address: 3466 Physical address: 54154 Value: 3
Virtual address: 38185 Physical address: 54313 Value: 0
Virtual address: 47127 Physical address: 23319 Value: 5
Virtual address: 16760 Physical address: 54648 Value: 0
Virtual address: 28977 Physical address: 37681 Value: 0
Virtual address: 6092 Physical address: 9932 Value: 0
Virtual address: 37300 Physical address: 4788 Value: 0
Virtual address: 31281 Physical address: 54833 Value: 0
Virtual address: 14555 Physical address: 43227 Value: 54
Virtual address: 21671 Physical address: 50343 Value: 41
Virtual address: 29921 Physical address: 14561 Value: 0
Virtual address: 4796 Physical address: 35260 Value: 0
Virtual address: 46655 Physical address: 43839 Value: -113
Virtual address: 24440 Physical address: 34680 Value: 0
Virtual address: 58885 Physical address: 45573 Value: 0
Virtual address: 49677 Physical address: 55053 Value: 0
Virtual address: 63906 Physical address: 41634 Value: 62
Virtual address: 24692 Physical address: 44404 Value: 0
Virtual address: 53968 Physical address: 18640 Value: 0
Virtual address: 21680 Physical address: 50352 Value: 0
Virtual address: 42910 Physical address: 29598 Value: 41
Virtual address: 33586 Physical address: 6194 Value: 32
Virtual address: 9541 Physical address: 39749 Value: 0
Virtual address: 59799 Physical address: 4503 Value: 101
Virtual address: 51566 Physical address: 48238 Value: 50
Virtual address: 22899 Physical address: 20339 Value: 92
Virtual address: 7642 Physical address: 21978 Value: 7
Virtual address: 8969 Physical address: 12809 Value: 0
Virtual address: 61612 Physical address: 4012 Value: 0
Virtual address: 14542 Physical address: 43214 Value: 14
Virtual address: 468 Physical address: 53716 Value: 0
Virtual address: 50595 Physical address: 55459 Value: 104
Virtual address: 2333 Physical address: 36637 Value: 0
Virtual address: 58085 Physical address: 21477 Value: 0
Virtual address: 51177 Physical address: 7145 Value: 0
Virtual address: 35854 Physical address: 55566 Value: 35
Virtual address: 41599 Physical address: 19583 Value: -97
Virtual address: 55685 Physical address: 23941 Value: 0
Virtual address: 10048 Physical address: 9536 Value: 0
Virtual address: 46318 Physical address: 17646 Value: 45
Virtual address: 23789 Physical address: 22509 Value: 0
Virtual address: 56542 Physical address: 56030 Value: 55
Virtual address: 11851 Physical address: 40779 Value: -110
Virtual address: 4441 Physical address: 56153 Value: 0
Virtual address: 48034 Physical address: 46754 Value: 46
Virtual address: 55853 Physical address: 56365 Value: 0
Virtual address: 17870 Physical address: 39630 Value: 17
Virtual address: 3696 Physical address: 48496 Value: 0
Virtual address: 43947 Physical address: 40363 Value: -22
Virtual address: 63797 Physical address: 41525 Value: 0
Virtual address: 11346 Physical address: 22610 Value: 11
Virtual address: 55231 Physical address: 24255 Value: -17
Virtual address: 8667 Physical address: 51163 Value: 118
Virtual address: 44441 Physical address: 45465 Value: 0
Virtual address: 44408 Physical address: 45432 Value: 0
Virtual address: 37102 Physical address: 3566 Value: 36
Virtual address: 22436 Physical address: 49828 Value: 0
Virtual address: 30161 Physical address: 41169 Value: 0
Virtual address: 23183 Physical address: 15759 Value: -93
Virtual address: 20250 Physical address: 14106 Value: 19
Virtual address: 24693 Physical address: 44405 Value: 0
Virtual address: 20395 Physical address: 14251 Value: -22
Virtual address: 22939 Physical address: 20379 Value: 102
Virtual address: 23414 Physical address: 42102 Value: 22
Virtual address: 7299 Physical address: 56707 Value: 32
Virtual address: 27911 Physical address: 56839 Value: 65
Virtual address: 64246 Physical address: 39414 Value: 62
Virtual address: 17120 Physical address: 29408 Value: 0
Virtual address: 57800 Physical address: 57288 Value: 0
Number of Translated Addresses = 500
Page Faults = 224
Page Fault Rate = 0.448
TLB Hits = 16
TLB Hit Rate = 0.032
